brig/cmd/brig/commands/build_delete.go

```diff
@@ -0,0 +1,42 @@
+package commands
+
+import (
+	"errors"
+	"io"
+
+	"github.com/Azure/brigade/pkg/storage"
+	"github.com/Azure/brigade/pkg/storage/kube"
+
+	"github.com/spf13/cobra"
+)
+
+const buildDeleteUsage = `Deletes a build, regardless of its state.`
+
+func init() {
+	build.AddCommand(buildDelete)
+}
+
+var buildDelete = &cobra.Command{
+	Use:   "delete BUILD_ID",
+	Short: "deletes build",
+	Long:  buildDeleteUsage,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		if len(args) == 0 {
```

Comment:
For future reference, here you could also use the following for the `cobra.Command`:

```
		Args:  cobra.ExactArgs(1),
```

Reply:
updated cobra and it works, thanks!